vertical-pod-autoscaler/examples/hamster-deprecated.yaml

```diff
@@ -11,6 +13,7 @@ spec:
   selector:
     matchLabels:
       app: hamster
+
```

Comment:
Is the blank line intentional?

Reply:
I added that to get a visual seperate between two items